params/config.go

```diff
@@ -165,7 +165,7 @@ var (
 			BlockReward:       big.NewInt(1e18),
 			FirstHalvingBlock: big.NewInt(53_557_371),
 			HalvingPeriod:     big.NewInt(63_115_200),
-			FinishRewardBlock: big.NewInt(1_000_000_000), // TODO fix last reward block
+			FinishRewardBlock: big.NewInt(2_000_000_000), // TODO fix last reward block
```

Comment:
this is a sample value but for test, I adjust this to a value greater than `FirstHalvingBlock + HalvingPeriod * HalvingTimes`